CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 - Change: [#3258] Current currency dropdown is now disabled when in the title screen.
 - Fix: [#3232] Crash when moving vehicle window when a message is showing a vehicle.
 - Fix: [#3256] Preferred currency not being loaded from config.
+- Fix: [#3262] Editing keyboard shortcut always modifies the first shortcut instead of the selected one.
```

Comment:
This isn't a fix from 25.08 its a fix from master when you broke it earlier this month.

Reply:
Oh, I'll revert that.